uniffi_bindgen/src/bindings/kotlin/gen_kotlin/mod.rs

```diff
@@ -219,10 +241,10 @@ impl KotlinCodeOracle {
                 Box::new(compounds::MapCodeType::new(inner, outer))
             }
             Type::External { .. } => panic!("no support for external types yet"),
-            Type::Wrapped { name, prim } => Box::new(wrapped::WrappedCodeType::new(
-                name,
-                self.create_code_type(prim.as_ref().clone()),
-            )),
+            Type::Wrapped { name, inner } => {
+                let config = self.config.wrappers.get(&name).cloned();
+                Box::new(wrapped::WrappedCodeType::new(name, *inner, config))
```

Comment:
My goal here is to use the FfiConverter class so that the lift/lower/read/write functions are always in the same place.  This would be one way to implement external types.  But it's kind of weird having only this code follow that pattern and I'm actually not sure how external types are going to shake out.

In other words, yeah this can just be `self.find(inner)` :)